dvclive/metrics.py

```diff
@@ -29,7 +29,7 @@ def __init__(
         self._step: int = 0
         self._html: bool = html
         self._summary = summary
-        self._metrics: Dict[str, float] = OrderedDict()
+        self._metrics: Dict[str, Tuple[float, float]] = OrderedDict()
```

Comment:
What's the reason for this change?

Reply:
It shouldn't had been pushed. I was considering altering `self._metrics` structure to handle the implicit `next_step` call in `dvclive.log`.

Reply:
Is `Dict[str, float]` always correct? I thought that `self._metrics` could be a nested dict?

Reply:
Yep, the typing is not correct. Recursive type hint is kind of a debatable topic. In addition, we should be probably using `typing.OrderedDict` . I guess `OrderedDict[str, Any]` would be the "proper" way?